src/common/components/github_tools.py

```
@@ -197,6 +205,61 @@
+
+            response = requests.get(
+                logs_url, headers={
+                    "Accept": "application/vnd.github+text",
+                    "Authorization": "Bearer " + auth_token,
+                }
+            )
+            response.raise_for_status()
+
+            # generate a random temp dir for extracting the logs (which is a zip file)
+            dir = tempfile.TemporaryDirectory()
+            dirname = f"{dir.name}/workflow_run_logs.zip"
```

Ruff (0.11.9)

Local variable `dirname` is assigned to but never used

Remove assignment to unused variable `dirname`

(F841)